### PR TITLE
Updated content on page

### PR DESCRIPTION
**Pull request by kierenhyl**

### CODE REVIEW

File: content/projects/open-alias/index.md [hunk 1]
```diff
@@ -27,49 +27,67 @@ font_color: "#fff"
...
-<br /><br />
+### 1. Register your name
 
+Register a name you like and the suffix `@nav.community` will be added automatically.
 
-### Who created OpenAlias?
+### 2. Copy message to console
 
-OpenAlias was created by contributors to the [Monero](https://getmonero.org/) cryptocurrency project who wanted to find a way of simplifying cryptocurrency payments by bridging the knowledge gap between cryptographically secure systems and users who aren't sophisticated developers.
+A line of code will be generated to assign the address to the alias. Follow the instruction to copy and paste the code to the console in your NavCoin Wallet's Debug window.
```
Comment (matt-auckland) [inline content/projects/open-alias/index.md]: "assign the address to the alias" -> "associate the address with the alias"

Is more technically correct

File: content/projects/open-alias/index.md [hunk 2]
```diff
@@ -27,49 +27,67 @@ font_color: "#fff"
...
 
-<br />
-<br />
+### 3. Verify
+
+The console in the Debug window will output a line of verification code. This code then needs to be copied back to the NavCoin OpenAlias to complete the verification process.
+
+## Why use OpenAlias?
+
+Open Alias was created by contributors to the [Monero](https://getmonero.org/) cryptocurrency project who wanted to find a way of simplifying cryptocurrency payments by bridging the knowledge gap between cryptographically secure systems and users who aren’t sophisticated developers.
```
Comment (matt-auckland) [inline content/projects/open-alias/index.md]: OpenAlias should have no space between 'Open' and 'Alias'

File: content/projects/open-alias/index.md [hunk 3]
```diff
@@ -27,49 +27,67 @@ font_color: "#fff"
...
+
+1. **It's easy** - You can transform long alphanumeric codes into an easily memorable address.
+
+2. **It's secure** - NavCoin Open Alias leverages DNSSEC to prevent MITM attacks on the alias.
```
Comment (matt-auckland) [inline content/projects/open-alias/index.md]: "Open Alias" -> "OpenAlias"

File: content/projects/open-alias/index.md [hunk 4]
```diff
@@ -27,49 +27,67 @@ font_color: "#fff"
...
-### Overview
 
-NAV OpenAlias is a little like PayPal, which lets you send a payment to an email address. The difference is that OpenAlias uses an easy to remember 'email style' address to represent your full NavCoin wallet address.
+## Overview
+NAV OpenAlias is a little like PayPal, which lets you send a payment to an email address. The difference is that OpenAlias uses an easy to remember ‘email style’ address to represent your full NavCoin wallet address. 
 
 For more information, visit [openalias.org](https://openalias.org/)
-<br /><br />
 
+## Registering for an NavCoin Open Alias
```
Comment (matt-auckland) [inline content/projects/open-alias/index.md]: "Open Alias" -> "OpenAlias"

**Comment by LauraHarris1** [comment]
Typo:

Registering for an NavCoin Open Alias

Should be "a" instead of "an"

File: content/projects/open-alias/index.md (hunk 1 above)
Comment (matt-auckland) [reply content/projects/open-alias/index.md]: I thought about this a bit more.

This step is only to get the signed message to verify that you own the address, it's the third step that actually associates the address with the alias

**Comment by zanuka** [comment]
the header, breadcrumb, and left menu still say Openalias instead of OpenAlias. 

screen shot 2018-08-28 at 1 57 38 pm

File: content/projects/open-alias/index.md [hunk 5]
```diff
@@ -19,57 +19,75 @@ show_on_projects: true
...
 percent_complete: 100
 github_url: "https://github.com/NAVCoin/navcoin-core/pull/213"
-project_url: "http://openalias.nav.community"
+project_url: "https://openalias.nav.community"
 reddit_url: ""
 twitter_url: ""
 card_color: "#7d5ab5"
 font_color: "#fff"
 weight: 100
 ---
```
Comment (matt-auckland) [inline content/projects/open-alias/index.md]: image

Currently shows like this on the projects page, you need to edit lines 15 and 16 in index.md so they say this.
```
completed: true
beta_testing: false
```

Comment (matt-auckland) [reply content/projects/open-alias/index.md]: Unless we are still considering this as being in beta

**Pull request merged**